app/livechat/server/sendMessageBySMS.js

```diff
@@ -42,5 +43,5 @@ callbacks.add('afterSaveMessage', function(message, room) {
 
 	SMSService.send(room.sms.from, visitor.phone[0].phoneNumber, message.msg);
 
-	return message;
+	return normalizeMessageAttachments(message);
```

Comment:
Do you mind of checking the `message.file` before returning? I think we should always return the `message` instead of the `normalizeMessageAttachments`.